app/views/teams/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with model: team, local: true do |form| %>
+  <div class="form-group">
+    <%= form.label :name %>
+    <%= form.text_field :name, minlength: 5, maxlength: 40, class: 'form-control' %>
+  </div>
+
+  <div class="submit-field">
```

Comment:
Is this a bootstrap class? If not, I think we can remove it, since it's not being used for any styles nor frontend logic.